src/renderer/components/ft-profile-edit/ft-profile-edit.vue

```diff
@@ -11,6 +11,17 @@
           @input="e => profileName = e"
         />
       </ft-flex-box>
+      <h3>{{ $t("Profile.Upload Profile Picture") }}</h3>
+      <ft-flex-box>
+        <ft-button>
+          <p>{{ $t("Profile.Choose an Image") }}</p>
+          <input
+            type="file"
+            class="default-upload"
+            @change="profileImageUpload"
+          />
```

Comment:
Please fix the linting issue
```suggestion
          >
```